libraries/rush-lib/src/logic/installManager/InstallHelpers.ts

```diff
@@ -71,6 +72,18 @@ export class InstallHelpers {
       }
 
       if (pnpmOptions.globalIgnoredOptionalDependencies) {
+        if (rushConfiguration.rushConfigurationJson.pnpmVersion !== undefined && semver.lt(rushConfiguration.rushConfigurationJson.pnpmVersion, '9.0.0')) {
+          // eslint-disable-next-line no-console
+          console.warn(
+            Colorize.yellow(
+              `Your version of pnpm ${rushConfiguration.rushConfigurationJson.pnpmVersion} ` +
+                `doesn't support "globalIgnoredOptionalDependencies". Please consider upgrading the ` +
+                `"pnpmVersion" setting in ${RushConstants.rushJsonFilename} to 9.0.0 or higher ` +
+                Colorize.bold('(please note that pnpm 9+ is not officially supported yet at the moment)')
```

Comment:
```suggestion
              `Your version of pnpm (${rushConfiguration.rushConfigurationJson.pnpmVersion}) ` +
                `doesn't support the "globalIgnoredOptionalDependencies" field in ${RushConstants.<whatever this is>}/${RushConstants.<pnpm-config.json constant>}. ` +
                `Remove this field or upgrade to pnpm 9.`
```

I don't think it's necessary to mention that pnpm 9 isn't officially supported yet in this particular error message.